mint/install-chrome/mint-leaf.yml

```diff
@@ -0,0 +1,132 @@
+      EOF
+        exit 1
+      fi
+
+      resolved_version=$(echo "$chrome" | jq -r '.version')
+      # restrict usage to Chrome >= 115
+      if echo "$resolved_version" | awk -F . '{ if ($1 >= 115) { exit 0 } else { exit 1 } }'; then
+        echo "${resolved_version}" > "${MINT_VALUES}/chrome-version"
+        echo "${chrome}" | jq -r '[.downloads.chrome[] | select(.platform == "linux64")] | last.url' > "${MINT_VALUES}/chrome-download-url"
+        echo "${chrome}" | jq -r '[.downloads.chromedriver[] | select(.platform == "linux64")] | last.url' > "${MINT_VALUES}/chromedriver-download-url"
+      else
+        cat << EOF > $(mktemp "${MINT_ERRORS}/error-XXXX")
+      The \`chrome-version\` provided is less than 115. Only versions 115 and higher are supported.
+      EOF
+        exit 1
+      fi
+    env:
+      CHROME_VERSION: ${{ params.chrome-version }}
+    cache: ${{ params.chrome-version =~ '^\d+\.\d+\.\d+\.\d+$' }}
+
+  - key: install
+    run: |
+      path_additions=""
+      echo "${CHROME_VERSION}" > "${MINT_VALUES}/chrome-version"
+
+      echo "Installing Chrome ${CHROME_VERSION}"
+      echo
+
+      curl -fsSL "${CHROME_DOWNLOAD_URL}" -o chrome.zip
+      sudo unzip chrome.zip -d "${WITHIN_DIRECTORY}"
+      chrome_dir="${WITHIN_DIRECTORY}/chrome-linux64"
+      echo "${chrome_dir}" > "${MINT_VALUES}/chrome-directory"
+      echo "${chrome_dir}/chrome" > "${MINT_VALUES}/chrome-binary"
+      if [[ "${ADD_TO_PATH}" == "true" ]]; then
+        path_additions="${chrome_dir}"
+      fi
+
+      if [[ "${INSTALL_CHROMEDRIVER}" == "true" ]]; then
+        echo "Installing Chromedriver for Chrome ${CHROME_VERSION}"
+        echo
+
+        curl -fsSL "${CHROMEDRIVER_DOWNLOAD_URL}" -o chromedriver.zip
+        sudo unzip chromedriver.zip -d "${WITHIN_DIRECTORY}"
+        chromedriver_dir="${WITHIN_DIRECTORY}/chromedriver-linux64"
+        echo "${chromedriver_dir}" > "${MINT_VALUES}/chromedriver-directory"
+        echo "${chromedriver_dir}/chromedriver" > "${MINT_VALUES}/chromedriver-binary"
+        if [[ "${ADD_TO_PATH}" == "true" ]]; then
+          path_additions="${path_additions}:${chromedriver_dir}"
+        fi
+      fi
+
+      if [[ -n "${path_additions}" ]]; then
+        echo "Adding ${path_additions} to PATH"
+        echo "${path_additions}" >> "${MINT_ENV}/PATH"
+      fi
+
+      echo "Installing Chrome system dependencies"
+      sudo apt-get update
+      sudo apt-get install \
+        libglib2.0-0 \
+        libgconf-2-4 \
+        libatk1.0-0 \
+        libatk-bridge2.0-0 \
+        libgdk-pixbuf2.0-0 \
+        libgtk-3-0 \
+        libgbm-dev \
+        libnss3-dev \
+        libxss-dev \
+        libasound2 \
+        xvfb \
+        fonts-liberation \
+        libu2f-udev \
+        xdg-utils
+      sudo apt-get clean
+
+      rm -f chrome.zip
+      rm -f chromedriver.zip
```

Comment:
Is there a --version flag or something we can pass to the binary to execute it here just to make sure it works?

Reply:
Yep! We use it in the tests

Reply:
I think we should do it here too (and probably just echo something like "Installed Chrome v....". Since the user is providing the directory to install it to, it's not unreasonable that in some cases the binary will be broken, I think.

Reply:
Yea, we have a number of echos already but I can add a final one